glos_qartod/qc.py

```diff
@@ -235,13 +297,21 @@ def apply_qc(self, ncvariable):
             test_params['arr'] = values
 
         if values.size > 0:
-           qc_flags = qc_tests[qartod_test](**test_params)
+            # Try to run the test.  If it fails, return an exception
+            try:
+                qc_flags = qc_tests[qartod_test](**test_params)
+            except Exception as e:
+                get_logger().error("QARTOD test application failed: %s", str(e))
+                return
+
+        # TODO: Ideally should handle zero length at the QARTOD level
```

Comment:
Please create an issue in this repository describing this and where it is and then remove it from the code.